### ISSUE TITLE
Setting Primary fails

### ISSUE THREAD

**Issue by davidgs** [microsoft/codetour#239]
In starting out with this stuff I was super-excited and probably over-eager. So I started playing around before reading **all** of the directions. 

I created a Tour of one part of my codebase, but then thought I'd like to add more tours. 

So I created another tour, and tried to use the `set primary` menu item.  But nothing changes. So I went into the json and remove the `primary` from the first tour, and added it to the second tour. No change. So I added the numbering convention. No change. 

![Screen Shot 2022-05-20 at 10 07 57 AM](https://user-images.githubusercontent.com/2071898/169546020-4fcc9bea-e590-4b49-907f-78867104fa38.png)

![Screen Shot 2022-05-20 at 10 08 11 AM](https://user-images.githubusercontent.com/2071898/169546094-e20919e8-416d-4768-930b-04898f89fe0f.png)

![Screen Shot 2022-05-20 at 10 08 25 AM](https://user-images.githubusercontent.com/2071898/169546138-3cd3c560-d58c-4441-9194-932ab4ed26f7.png)

Now, I did export the first tour to a Gist yesterday, so I wonder if that is preventing the changes?

**Comment by lostintangent** [comment]
Yeah something must have regressed here 🤔 Thanks for reporting this!

**Comment by lostintangent** [comment]
This should now be fixed
